packages/ui-components/src/components/Matrix/MatrixCell.tsx

```diff
@@ -150,11 +164,18 @@ export const MatrixCell = ({ value, rowTitle, isCategory, colKey }: MatrixCellPr
       />
     );
 
-  const displayValue = value === undefined || value === null ? '—' /* em dash */ : value;
+  const displayValue = value ?? '—'; // em dash
```

Comment:
Just checking how this handles `0` or `false` values - in those cases we would want the actual value to show because they are useful to the user

Reply:
[`??` specifically checks for `undefined` or `null`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), so falsy values like `0` will still work the way we want. This refactor has identical behaviour

Fair question though—thanks for the rigour!